tcms/rpc/tests/test_testexecution.py

```diff
@@ -334,6 +334,34 @@ def test_update_with_non_existing_assignee_username(self):
                 "assignee": 'nonExistentUsername'
             })
 
+    def test_update_with_numeric_case_text_version(self):
+        self._update_case_text_version()
+        latest_case_text_version = self.case_run_1.case.history.latest().history_id
+
+        execution = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
+            "case_text_version": str(latest_case_text_version)
+        })
+        self.assertEqual(execution["case_text_version"], latest_case_text_version)
+
+    def test_update_with_latest_case_text_version(self):
+        self._update_case_text_version()
```

Comment:
`initial_case_text_version = self.case_run_1.case_text_version` would come in before the update here and then after the API call you can assert against the latest text version.